_posts/2017-03-31-traefik-load-balancing.markdown

```diff
@@ -0,0 +1,437 @@
+---
+layout: post
+title:  "WebApps with Traefik LoadBalancing"
+date:   2017-03-31 10:51:47 +0530
+author: "Sébastien Allamand (allamand)"
+categories: webapps
```

Comment:
`categories` should be `category` and needs to be one of ["beginner", "intermediate", "advanced"]